pygments/lexers/wren.py

```diff
@@ -0,0 +1,97 @@
+"""
+    pygments.lexers.wren
+    ~~~~~~~~~~~~~~~~~~~~
+    Lexer for Wren.
+    :copyright: Copyright 2006-2022 by the Pygments team, see AUTHORS.
+    :license: BSD, see LICENSE for details.
+"""
+
+import re
+
+from pygments.lexer import RegexLexer
+from pygments.token import Whitespace, Punctuation, Keyword, Name, Comment, \
+    Operator, Number, String
+
+__all__ = ['WrenLexer']
+
+class WrenLexer(RegexLexer):
+    name = 'Wren'
+    aliases = ['wren']
+    filenames = ['*.wren']
+
```

Comment:
Please add an `url` as well which points to the language website, see: https://github.com/pygments/pygments/blob/master/pygments/lexer.py#L81

Reply:
Done.